extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -714,7 +734,11 @@ export class AmpA4A extends AMP.BaseElement {
         layoutAdPromiseDelay: Math.round(delta),
         isAmpCreative: !!creativeMetaData,
       });
-      if (creativeMetaData) {
+      if (!creativeMetaData) {
+        // Non-AMP creative case, will verify ad url existence.
+        return this.renderNonAmpCreative_();
+      }
+      if (!creativeMetaData.collapse) {
```

Comment:
I think you're missing the `creativeMetaData && creativeMetaData.collapse` case here?  There should be some return value when `creativeMetaData.collapse == true`.  (I guess in the case of no explicit return, there's an implicit promise return with no arg, but it would be good to have an explicit one regardless.)